server/bin/pbench-tarball-stats.py

```diff
@@ -175,10 +181,21 @@ def main(options):
     print(f"{invalid_cnt:18n} bad tar balls, total count")
     print(f"{invalid_size:18n} bad tar balls, total size")
 
-    print(
-        f"{len(good.keys()):18n} controllers with good tar balls (EC2 {ec2_cnt:n}, RH {rh_cnt:n})"
-    )
-    print(f"{good_cnt:18n} good tar balls, total count")
+    print(f"{len(good.keys()):18n} controllers with good tar balls (", end="")
+    satellites_ctrl = collections.Counter()
+    for good_ctrl in good.keys():
+        satellites_ctrl[get_sat_name(good_ctrl)] += 1
+    first = True
+    for name, total in sorted(satellites_ctrl.items()):
+        comma, first = ("", False) if first else (", ", False)
+        print(f"{comma}{name} {total:n}", end="")
```

Comment:
I think there's a more Pythonic way of doing this.  Perhaps something like,
```
    cnt_strs = [ f"{name} {total:n}" for name, total in sorted(satellites_ctrl.items())]
    print(", ".join(cnt_strs), end="")
```

Ditto for 194-197.